### DISCUSSION TITLE
Spaces in source code paths

### DISCUSSION THREAD

**Discussion by gutley** [mozilla-mobile/firefox-ios#13271] [General]
Whilst perusing the entire source tree today I noticed that there is one directory that has spaces in its name, whereas I think all the others have no spaces between words.

`Client/Frontend/Login Management/`

I only mention it as this is the sort of thing that can trip up shell scripts and the like.

**Comment by lmarceau** [comment]
We can definitely remove that space so this aligns with other folders. Do you want to open a PR for this (and a ticket to link to) ?

**Reply by gutley** [reply]
@lmarceau PR now available: https://github.com/mozilla-mobile/firefox-ios/pull/13315

Note there is one folder that I left alone in this - `Shared/Supporting Files`, which contains all the localisation stuff. I felt it was too risky for anyone outside of Mozilla to change that as I suspect you have a bunch of scripts that expect that folder name when generating new strings and string files etc.

I also didn't attempt to rename any files or folders within asset catalogs. Too tricky to check that they are all still working properly.

BTW -  there is one really weird image file that might need investigating as it has a space at the start of its name:
`Client/Frontend/Menu/Menu.xcassets/Image.imageset/ Switch-On@2x.png`

FYI - To list all of the paths in the entire folder with spaces:
`find . | grep " "`
